src/en/charms-constraints.md

```diff
@@ -139,6 +142,13 @@ that model, with the `set-model-constraints` command:
 juju set-model-constraints mem=4G
 ```
 
+For the LXD cloud, all new machines in the current model will be limited
+to an instance type of 'c5.large' (2 CPU and 4 GiB):
+
+```bash
+juju set-model-constraints instance-type=c5.xlarge
```

Comment:
lxd users have the instance type available to them but it's a 1% use case in our experience. Let's demo the hard cpu/etc settings in examples unless talking about the instance-type feature.

Reply:
I don't follow. This **is** showing the instance-type feature.

Reply:
This seems to be highlighting using set-model-constraints and not explaining how instance types is a valid constraint key to be used in all clouds.

Reply:
I'm merely showing how to set the instance-type constraint for a LXD model. That's all.

Also, the instance-type constraint is not supported by all clouds.